rllab/envs/mujoco/dynamics_randomization/__init__.py

```diff
@@ -0,0 +1,4 @@
+from rllab.envs.mujoco.dynamics_randomization.randomized_env import randomize
```

Comment:
Please name the package `randomization`, since it's not actually specific to dynamics.